docs/README.md

```diff
@@ -49,6 +49,66 @@ accelerate launch  main.py \
 
 If you want to evaluate only on the first $n$ samples instead of all the test dataset, set `limit` argument to $n$. 
 
+
+### HumanEvalPack
```

Comment:
Could you add some instructions to run HumanEvalPack for OpenAI models with `humanevalpack_openai` task?